app/domain/authentication/authn_jwt/validate_status.rb

```diff
@@ -0,0 +1,123 @@
+module Authentication
+  module AuthnJwt
+
+    ValidateStatus = CommandClass.new(
+      dependencies: {
+        create_signing_key: Authentication::AuthnJwt::SigningKey::CreateSigningKeyFactory.new,
+        fetch_issuer_value: Authentication::AuthnJwt::ValidateAndDecode::FetchIssuerValue.new,
+        fetch_identity_from_token: Authentication::AuthnJwt::IdentityProviders::IdentityFromDecodedTokenProvider,
```

Comment:
change `fetch_identity_from_token` to `identity_from_decoded_token_provider`

Reply:
Done